src/main/java/org/shanerx/tradeshop/Utils.java

```diff
@@ -21,7 +21,11 @@
 
 package org.shanerx.tradeshop;
 
-import org.bukkit.*;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.Nameable;
+import org.bukkit.OfflinePlayer;
```

Comment:
Good one, thought we had ironed out all wildcard imports.

Reply:
I didn't even touch that, so my idea auto corrected that